services/user.py

```diff
@@ -0,0 +1,59 @@
+from db.models import User
+
+
+def create_user(
+        username: str,
+        password: str,
+        email: str = None,
+        first_name: str = None,
+        last_name: str = None
+) -> User:
+    user = User.objects.create_user(

```

Comment:
Good practice - not "User", but get_user_model() (read in a checklist)

Reply:
Thank you! didn`t  pay attention to these things